challenge3.txt

```diff
@@ -0,0 +1,7 @@
+# the effect of graduating college on the minimum wage 
```

Comment:
Work on writing more descriptive commit messages. This will help in more complex future collaborations.

Reply:
2.75/3